### PR TITLE
Feature/dialogs

### PR DESCRIPTION
**Pull request by michsa**
- Popover (facelift)
- Dropdown (new thing, though really just a variant of Popover)
- Modal (facelift & new API: `Modal.Header`, `Modal.Content`, `Modal.Footer`)
- Tooltip (react-tooltip lib)

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>:exclamation: This PR is BIG (+403 -106)  
Please keep it below 500 net changes</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>The README has not been updated. Please update the README.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Could not find any browser results.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 9a9aa08b039e029094bd4e50d6b555626f40326d
</p>

File: src/Divider.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import theme from './theme'
+
+let Divider = ({ vertical = false, m = 1 }) => (
```
Comment (daedalus28) [inline src/Divider.js]: `m` is really vauge

File: src/Modal.js [hunk 2]
```diff
@@ -1,46 +1,96 @@
...
-        className={`default-modal-bg ${className}`}
       >
-        <div
-          style={{
-            backgroundColor: '#fff',
-            ...style,
-          }}
+        <Box
+          variant="modal"
+          p={2.5}
```
Comment (daedalus28) [inline src/Modal.js]: ?

File: src/Box.js [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
 import { jsx } from '@emotion/core'
 import _ from 'lodash/fp'
+import theme, { withPadding } from './theme'
 
 let Box = ({ as: As = 'div', variant = 'normal', ...props }) => (
   <As
-    css={theme => ({
-      borderRadius: theme.spaces.xs,
+    css={{
+      borderRadius: 3,
```
Comment (michsa) [inline src/Box.js]: this is being pulled into `theme` in the next pr

**Pull request merged**